src/mastercore_mdex.cpp

```diff
@@ -398,8 +421,10 @@ int mastercore::MetaDEx_ADD(const std::string& sender_addr, uint32_t prop, int64
             return METADEX_ERROR -70;
         } else {
             // move tokens into reserve
-            assert(update_tally_map(sender_addr, prop, -new_mdex.getAmountRemaining(), BALANCE));
-            assert(update_tally_map(sender_addr, prop, new_mdex.getAmountRemaining(), METADEX_RESERVE));
+            //assert(update_tally_map(sender_addr, prop, -new_mdex.getAmountRemaining(), BALANCE));
```

Comment:
Ahh yes sorry no longer needed to bypass them (I was bypassing them before your fixes went in to continue work on the UI).  Updated.